src/BuildScriptGenerator/PlatformInstallerBase.cs

```diff
@@ -97,7 +113,7 @@ public virtual void InstallPlatformSpecificSkeletonDependencies(StringBuilder st
             var snippet = new StringBuilder();
             snippet
                 .AppendLine()
-                .AppendLine($"if grep -q cli \"/opt/oryx/.imagetype\"; then")
+                .AppendLine($"if grep -q -e cli \"/opt/oryx/.imagetype\" -e jamstack \"/opt/oryx/.imagetype\"; then")
```

Comment:
nit: just to clarify (since I'm not too familiar with `grep`): this is installing skeleton dependencies if the `.imagetype` file contains _either_ `cli` or `jamstack`, correct?

Reply:
yep, that is correct.